<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WTC-5159473-l2" ignored="0" name="MFAC_Test" tests="33" failures="18" timestamp="04 Jan 2019 19:43:54 GMT" time="37.287" errors="0">
  <testcase name="SetEvironmentLevel" time="0.013" classname="SupportClasses.Set_Environment"/>
  <testcase name="VerifyPinVelocity" time="1.447" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;a8d881ff-b180-45f7-8e08-9a060d967cfc&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a8d881ff-b180-45f7-8e08-9a060d967cfc","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="VerifyPinVelocity" time="2.552" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;66db0677-6947-4613-bd2a-6fef7e185742&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"66db0677-6947-4613-bd2a-6fef7e185742","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="VerifyPinVelocity" time="3.788" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;e0d446a5-f243-4ce7-847f-110a1a84eb91&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"e0d446a5-f243-4ce7-847f-110a1a84eb91","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="VerifyPinVelocity" time="4.904" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;f7e4e7bb-c6bf-44ce-ada7-9e36eb623490&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"f7e4e7bb-c6bf-44ce-ada7-9e36eb623490","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="VerifyPinVelocity" time="6.138" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;87937ed3-9969-4a99-873c-bf68109f36a2&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"87937ed3-9969-4a99-873c-bf68109f36a2","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="VerifyPinVelocity" time="7.725" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;bf38a1a2-9d03-4f18-89d1-c5d07a931868&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"bf38a1a2-9d03-4f18-89d1-c5d07a931868","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinVelocity(MFAC.java:202)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinVelocity -->
  <testcase name="IssuePin" time="10.088" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;pinOTP&amp;quot; and a string containing &amp;quot;pinExpirationDate&amp;quot;)
     but: a string containing &amp;quot;pinOTP&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;6c69b9b2-f247-41e0-9f73-10b9f845feec&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"6c69b9b2-f247-41e0-9f73-10b9f845feec","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePin(MFAC.java:147)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- IssuePin -->
  <testcase name="VerifyPinNoLongerValid" time="11.588" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinOTP&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;f6ee3490-5518-4728-beea-225a91b07660&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"f6ee3490-5518-4728-beea-225a91b07660","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinNoLongerValid(MFAC.java:229)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinNoLongerValid -->
  <testcase name="IssuePin" time="12.678" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;pinOTP&amp;quot; and a string containing &amp;quot;pinExpirationDate&amp;quot;)
     but: a string containing &amp;quot;pinOTP&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;589d3ba9-5730-4772-8571-ce694011137f&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"589d3ba9-5730-4772-8571-ce694011137f","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePin(MFAC.java:147)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- IssuePin -->
  <testcase name="IssuePin" time="13.958" classname="MFAC.MFAC"/>
  <testcase name="IssuePinVelocity" time="15.56" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;pinOTP&amp;quot; and a string containing &amp;quot;pinExpirationDate&amp;quot;)
     but: a string containing &amp;quot;pinOTP&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;a8f9f8c7-3a9c-4cbb-b9ed-bd61df20e9e6&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"a8f9f8c7-3a9c-4cbb-b9ed-bd61df20e9e6","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePinVelocity(MFAC.java:165)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- IssuePinVelocity -->
  <testcase name="VerifyPinNoLongerValid" time="17.737" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinOTP&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;786d47e3-646a-4e89-9a38-02352dc1aeeb&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinOTP"
     but: was "{"transactionId":"786d47e3-646a-4e89-9a38-02352dc1aeeb","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinNoLongerValid(MFAC.java:229)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinNoLongerValid -->
  <testcase name="VerifyPinValid" time="19.465" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;a567db67-c812-4abc-add8-5aae177c9cb4&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"a567db67-c812-4abc-add8-5aae177c9cb4","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinValid(MFAC.java:184)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinValid -->
  <testcase name="VerifyPinValid" time="21.998" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;pinExpirationDate&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;30d56f57-e408-4956-b31c-08dcac7b3006&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "pinExpirationDate"
     but: was "{"transactionId":"30d56f57-e408-4956-b31c-08dcac7b3006","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.VerifyPinValid(MFAC.java:184)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- VerifyPinValid -->
  <testcase name="IssuePin" time="23.885" classname="MFAC.MFAC"/>
  <testcase name="IssuePinVelocity" time="24.969" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: (a string containing &amp;quot;pinOTP&amp;quot; and a string containing &amp;quot;pinExpirationDate&amp;quot;)
     but: a string containing &amp;quot;pinOTP&amp;quot; was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;2b62bd12-7701-45ee-9bdb-8b2cd0eecac0&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;5700&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, you have exceeded your attempts for verification. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: (a string containing "pinOTP" and a string containing "pinExpirationDate")
     but: a string containing "pinOTP" was "{"transactionId":"2b62bd12-7701-45ee-9bdb-8b2cd0eecac0","errors":[{"code":"5700","message":"Unfortunately, you have exceeded your attempts for verification. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.IssuePinVelocity(MFAC.java:165)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- IssuePinVelocity -->
  <testcase name="IssuePinVelocity" time="26.533" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinValid" time="27.909" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinValid" time="27.963" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinVelocity" time="30.995" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinVelocity" time="31.266" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinVelocity" time="32.692" classname="MFAC.MFAC"/>
  <testcase name="IssuePinVelocity" time="34.371" classname="MFAC.MFAC"/>
  <testcase name="VerifyPinVelocity" time="34.516" classname="MFAC.MFAC"/>
  <testcase name="AddressVelocity" time="34.642" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;ALLOW&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;6e02498e-e70f-45f5-a5bf-a5b3f30d634b&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;DENY&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"6e02498e-e70f-45f5-a5bf-a5b3f30d634b","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="VerifyPinNoLongerValid" time="34.798" classname="MFAC.MFAC"/>
  <testcase name="AddressVelocity" time="34.936" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;ALLOW&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;5a19df54-d08b-4117-87bd-a44d4476fbd9&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;DENY&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"5a19df54-d08b-4117-87bd-a44d4476fbd9","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="VerifyPinVelocity" time="35.18" classname="MFAC.MFAC"/>
  <testcase name="AddressVelocity" time="35.347" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;ALLOW&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;193faa29-2535-489a-8fe7-ddd05088900d&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;DENY&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"193faa29-2535-489a-8fe7-ddd05088900d","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="VerifyPinNoLongerValid" time="35.587" classname="MFAC.MFAC"/>
  <testcase name="AddressVelocity" time="35.747" classname="MFAC.MFAC">
    <failure type="java.lang.AssertionError" message="
Expected: a string containing &amp;quot;ALLOW&amp;quot;
     but: was &amp;quot;{&amp;quot;transactionId&amp;quot;:&amp;quot;835fc60d-3185-4ad7-a1a2-db8ff9874962&amp;quot;,&amp;quot;errors&amp;quot;:[{&amp;quot;code&amp;quot;:&amp;quot;DENY&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Unfortunately, too many failed attempts for registration have occurred. Please try again later.&amp;quot;}]}&amp;quot;">
      <![CDATA[java.lang.AssertionError: 
Expected: a string containing "ALLOW"
     but: was "{"transactionId":"835fc60d-3185-4ad7-a1a2-db8ff9874962","errors":[{"code":"DENY","message":"Unfortunately, too many failed attempts for registration have occurred. Please try again later."}]}"
at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
at org.junit.Assert.assertThat(Assert.java:956)
at org.junit.Assert.assertThat(Assert.java:923)
at MFAC.MFAC.AddressVelocity(MFAC.java:128)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- AddressVelocity -->
  <testcase name="VerifyPinVelocity" time="33.38" classname="MFAC.MFAC"/>
</testsuite> <!-- MFAC_Test -->
